src/scss/constants.scss

```
@@ -1,11 +1,16 @@
-$primaryColor: var(--primary-color);
-$backgroundColor: var(--background-color);
+$primaryColor: var(--primary-color-deprecated);
+
+$primary-accent--over: var(--primary-accent--over);
+$primary-accent--under: var(--primary-accent--under);
+$primary-content--over: var(--primary-content--over);
+$primary-content-under: var(--primary-content--under);
+
+$primary--darken-20pp: var(--primary--darken-20pp);
+$primary--lighten-10pp: var(--primary--lighten-10pp);
```

Main colors: primary, secondary, accent ...
I don't think there will be `primary-accent` and `secondary-accent`...

Example from Material Design:
>A theme is a collection of color and typography options. Each theme includes three palettes that determine component colors:
>  - A primary palette for the color that appears most frequently throughout your application
>  - An accent, or secondary, palette used to selectively highlight key parts of your UI
>  - A warn, or error, palette used for warnings and error states

```suggestion
$accent--over: var(--primary-accent--over);
$accent--under: var(--primary-accent--under);
$content--over: var(--primary-content--over);
$content--under: var(--primary-content--under);

$primary--darken-20pp: var(--primary--darken-20pp);
$primary--lighten-10pp: var(--primary--lighten-10pp);
```

---

fixed in be7e3130da6a0e0b2b73a6e280423b17a7df9052